vendor-extra/LiberoContentBundle/tests/ViewConverter/FrontContentHeaderVisitorTest.php

```diff
@@ -153,7 +130,7 @@ function (Element $object, ?string $template, array $context) : View {
         $this->assertEquals(
             [
                 'contentTitle' => [
-                    'object' => $element->childNodes->item(0),
+                    'element' => '/libero:front/libero:title',
```

Comment:
To clarify, it's a bit confusing that this format (`element` rather than `object`) it's part of the test cases through `ViewConvertingTestCase`, as reading it in the expected field I was expecting it to have changed somewhere in the production code.

Reply:
I would rename `createConverter` to something more specific like `createDumpingConverter` or similar, since it introduces the custom code that we rely upon in these assertions